utils/cache.py

```
@@ -0,0 +1,217 @@
+        user_cache.set("key", item)
+
+    # Bounded cache with LRU eviction (max 1000 entries)
+    bounded: TTLCache[str, bytes] = TTLCache(ttl=60, maxsize=1000)
+"""
+
+from __future__ import annotations
+
+import time
+from collections import OrderedDict
+from collections.abc import Awaitable, Callable
+from typing import Any, Generic, TypeVar
```

Flake8 (7.3.0)

[error] 'typing.Any' imported but unused

(F401)